testsuite/run_sets/uyuni.yml

```diff
@@ -0,0 +1,141 @@
+- features/core_srv_create_repository.feature
+- features/core_srv_users.feature
+- features/core_srv_create_activationkey.feature
+- features/core_srv_osimage_profiles.feature
+- features/core_srv_docker_profiles.feature
+# initialize SUSE Manager proxy
+  # one of: core_proxy_register_as_trad_with_script.feature
+  #         core_proxy_register_as_minion_with_script.feature
+  #         core_proxy_register_as_minion_with_gui.feature
+- features/core_proxy_register_as_minion_with_gui.feature
+- features/core_proxy_branch_network.feature
+# initialize clients
+- features/core_trad_register_client.feature
+- features/core_min_bootstrap.feature
+- features/core_min_salt_ssh.feature
+- features/core_centos_salt_ssh.feature
+- features/core_ubuntu_salt_ssh.feature
+# these features sync real channels (last core features)
+- features/core_srv_sync_channels.feature
+- features/core_srv_setup_wizard.feature
+
+## Core features END ###
+
+
+## Secondary features BEGIN ##
+
+# IDEMPOTENT
+
+- features/srv_menu.feature
+- features/allcli_reboot.feature
+- features/trad_config_channel.feature
+- features/trad_lock_packages.feature
+- features/min_centos_salt.feature
+- features/trad_centos_client.feature
+- features/minssh_centos_salt_install_package_and_patch.feature
+- features/min_ubuntu_salt.feature
+- features/min_ubuntu_salt_install_package.feature
+- features/min_bootstrap_xmlrpc.feature
+- features/minssh_bootstrap_xmlrpc.feature
+- features/min_bootstrap_script.feature
+- features/min_activationkey.feature
+- features/trad_migrate_to_minion.feature
+- features/trad_migrate_to_sshminion.feature
+- features/trad_need_reboot.feature
+- features/trad_ssh_push.feature
+- features/srv_change_password.feature
+- features/srv_check_sync_source_packages.feature
+- features/min_salt_software_states.feature
+- features/min_docker_xmlrpc.feature
+- features/min_docker_build_image.feature
+- features/trad_metadata_check.feature
+- features/srv_clone_channel_npn.feature
+- features/trad_cve_id_new_syntax.feature
+- features/trad_weak_deps.feature
+- features/srv_nagios.feature
+- features/srv_cve_audit.feature
+- features/min_salt_install_with_staging.feature
+- features/srv_xmlrpc_activationkey.feature
+- features/allcli_overview_systems_details.feature
+- features/srv_distro_cobbler.feature
+- features/srv_mainpage.feature
+- features/srv_xmlrpc_user.feature
+- features/srv_salt_download_endpoint.feature
+- features/srv_virtual_host_manager.feature
+- features/trad_baremetal_discovery.feature
+- features/trad_action_chain.feature
+- features/min_action_chain.feature
+- features/minssh_action_chain.feature
+- features/min_salt_formulas.feature
+- features/min_salt_formulas_advanced.feature
+- features/min_docker_auth_registry.feature
+- features/srv_docker_advanced_content_management.feature
+- features/srv_docker_cve_audit.feature
+# OS image build tests are required for proxy_retail_pxeboot feature
+- features/min_osimage_build_image.feature
```

Comment:
You can also drop this one, it is only needed by retail and take a lot of time to build.